backend/src/schema/resolvers/posts.spec.js

```diff
@@ -462,9 +754,160 @@ describe('UpdatePost', () => {
           expected,
         )
       })
-    }) */
+    })
+
+    describe('change post type to event', () => {
+      describe('with missing event start date', () => {
+        it('throws an error', async () => {
+          await expect(
+            mutate({
+              mutation: updatePostMutation,
+              variables: { ...variables, postType: 'Event' },
+            }),
+          ).resolves.toMatchObject({
+            errors: [
+              {
+                message: "Cannot read properties of undefined (reading 'eventStart')",
+              },
```

Comment:
Is this the behaviour we want?

Reply:
Standard graphql schema error message. Yes.